### DISCUSSION TITLE
Allow custom HTML markup to be added easily

### DISCUSSION THREAD

**Discussion by hugo-sid** [hugo-sid/hugo-blog-awesome#152] [Feature requests]
Allow theme users to include custom HTML markup before the end of `<head>`
& before the end of `<body>`.

**Comment by jannes** [comment]
Are you still planning to add this? I could send a PR for this if you are open for it

**Reply by hugo-sid** [reply]
Yes. Please go ahead.

**Comment by hugo-sid** [comment]
This feature has been released in [v1.18.0](https://github.com/hugo-sid/hugo-blog-awesome/releases/tag/v1.18.0).
